shared/chat/inbox/row/divider/index.js

```diff
@@ -1,85 +1,105 @@
 // @flow
 import * as React from 'react'
-import {ClickableBox, Box, Text, Badge} from '../../../../common-adapters'
-import {globalStyles, globalColors, globalMargins, isMobile} from '../../../../styles'
+import {ClickableBox, Box2, Text, Badge} from '../../../../common-adapters'
+import {
+  styleSheetCreate,
+  collapseStyles,
+  globalStyles,
+  globalColors,
+  globalMargins,
+  isMobile,
+} from '../../../../styles'
+import * as RowSizes from '../sizes'
 
 type Props = {
   badgeCount: number,
   hiddenCount: number,
   style?: any,
-  showSmallTeamsExpandDivider: boolean,
+  showButton: boolean,
   toggle: () => void,
 }
 
 class Divider extends React.PureComponent<Props> {
   render() {
     return (
-      <Box style={this.props.style ? {..._toggleContainer, ...this.props.style} : _toggleContainer}>
-        {this.props.showSmallTeamsExpandDivider && (
-          <ClickableBox onClick={this.props.toggle} className="toggleButtonClass" style={_toggleButtonStyle}>
-            <Text type="BodySmallSemibold" style={_buttonTextStyle}>
+      <Box2
```

Comment:
What's the difference between `Box` and `Box2`? `Box2` seems to have some extra styling in it, but what was the motivation for the creation of `Box2`?

Reply:
Take a look at the stories for `Box` and `Box2` to understand a bit more about them: https://github.com/keybase/client/blob/master/shared/common-adapters/box.stories.js

Basically `Box` is just a `div` (on desktop) and a `View` (on mobile). `Box2` has a few props for injecting normal gaps, etc. Have a look at its props to know what's available: https://github.com/keybase/client/blob/master/shared/common-adapters/box.stories.js